Changes

```diff
@@ -1,5 +1,7 @@
 {{$NEXT}}
 
+    - Add Test2::Plugin::DebugOnFail
```

Comment:
Looks like this was the name of your previous attempt. I consider DataLine a better name.